README.md

```diff
@@ -1,26 +1,34 @@
 [![Build Status](https://travis-ci.org/stryker-mutator/stryker-cli.svg?branch=master)](https://travis-ci.org/stryker-mutator/stryker-cli)
 [![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-# Stryker Command-Line Interface
+![Stryker](https://github.com/stryker-mutator/stryker/raw/master/stryker-80x80.png)
 
+# Stryker Command-Line Interface
 This is the command-line interface (CLI) for [Stryker](https://stryker-mutator.github.io), the JavaScript mutation testing framework.
 
-# Installation
+If you plan on using Stryker in more projects, the Stryker-CLI is the easiest way to install, configure and run Stryker for your project.
 
-The Stryker CLI can be easily installed using NPM.
+# Installation
+The Stryker-CLI can be easily installed using NPM.
 
 ```
-> npm install -g stryker-cli
+$ npm install -g stryker-cli
 ```
```

Comment:
Maybe we can clarify this even more. Something like:

The Stryker-CLI installation does **NOT** install `stryker` itself. Instead, it only passes along commands to your local `stryker` (i.e. the stryker installation contained in your `node_modules` folder), or prompt to install a local stryker if none could be found. Therefore you should *always install it globally* with `-g` or `--global`. If you want to use stryker directly from [npm scripts](https://docs.npmjs.com/files/package.json#scripts), feel free to do so without stryker-cli.